doc/whats_new/upcoming_changes/sklearn.utils/30187.api.rst

```diff
@@ -0,0 +1,4 @@
+- :func:`utils.estimator_checks.check_estimator_sparse_tag` ensures that
```

Comment:
The filename is `30187.api.rst` but I think it should be considered a fix since those tag values were wrong/missing.

Reply:
This comment is now obsolete. Please refer to https://github.com/scikit-learn/scikit-learn/pull/30187#discussion_r1827900270 instead.